Packages/com.createneptune.sdk/Runtime/CNExtensions.cs

```diff
@@ -206,15 +205,46 @@ public static void CreateObjectPool(List<GameObject> gameObjectList, GameObject
             }
         }
 
-        public static T GetPooledObject<T>(List<GameObject> objectPool, GameObject gameObject,
-            int physicsLayer, Transform t, Vector3 positionOffset, Quaternion rotationOffset, bool ignoreParentTransform)
+        /// <summary>
+        /// Returns an object from the specified pool. Creates a new object and adds it to the pool if necessary.
+        /// </summary>
+        /// <param name="objectPool">The active pool</param>
+        /// <param name="gameObject">The prefab to be used to create a new object</param>
+        /// <param name="parent">The transform under which a new object should be placed</param>
+        /// <returns>An active object from the requested pool</returns>
+        private static GameObject GetPooledObject(List<GameObject> objectPool, GameObject gameObject, Transform parent)
```

Comment:
should this be public? might want to say if it is meant to be private that it's just a helper if that's the case

Reply:
This should also be marked as obsolete now, shouldn't it? It's being called by the obsolete functions below.

Reply:
also made this one public